src/otaclient/ota_core.py

```diff
@@ -171,8 +171,6 @@ def __init__(
         cookies_json: str,
         ca_chains_store: CAChainStore,
         upper_otaproxy: str | None = None,
-        boot_controller: BootControllerProtocol,
-        create_standby_cls: type[RebuildMode],
```

Comment:
These variables are set in `_OTAUpdater` class because slot setting is dedicated to data updates.